### ISSUE TITLE
notebook: autocomplete autocompletes twice

### ISSUE THREAD

**Issue by hiaselhans** [ipython/ipython#7996]
in ipython notebook 3 (yupiter) i encountered that autocomplete behaves a little wrong on some edge cases:

```
import matplotlib
matplotlib.b[tab]e[tab]
```

expected result

```
matplotlib.bezier
```

Actual result

```
matplotlib.bezierzier
```

**Comment by Carreau** [comment]
Closed a week ago in #7889.

**Comment by hiaselhans** [comment]
cool, thanks. sorry for the noise
